hippie.py

```diff
@@ -54,15 +68,20 @@ def on_init(self, views):
             index_view(view)
 
     def on_modified_async(self, view):
-        words_by_view[view] = None  # Drop cached word set
+        window = view.window()
+        if not window:
+            ...
+        else:
```

Comment:
Note: Since we use the `_async` variant, we can have detached views, and must check `window`.  We could just use the sync callback and then `view.window()` is safe.